packages/extension/src/serviceWorker/chromeListeners/index.ts

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { onBeforeSendHeadersListener } from './beforeSendHeadersListener';
+import { onResponseStartedListener } from './onResponseStartedListener';
+
+import './runtimeStartUpListener';
+import './runtimeOnInstalledListener';
+import './runtimeOnMessageListener';
+
+import './syncStorageOnChangedListener';
+
+import { onTabCreatedListener } from './tabOnCreatedListener';
+import './tabOnRemovedListener';
+import './tabsOnUpdatedListener';
+
+import './windowsOnRemovedListener';
+import './windowsOnCreatedListener';
+
+chrome.webRequest.onBeforeSendHeaders.addListener(
+  onBeforeSendHeadersListener,
+  { urls: ['*://*/*'] },
+  ['extraHeaders', 'requestHeaders']
+);
+
+chrome?.webRequest?.onResponseStarted?.addListener(
+  onResponseStartedListener,
+  { urls: ['*://*/*'] },
+  ['extraHeaders', 'responseHeaders']
+);
+
+chrome.tabs.onCreated.addListener(onTabCreatedListener);
```

Comment:
Other imports are addListener callbacks in their files.
Like syncStorage have addListener at the bottom of the file. 
We can follow that consistency.

Reply:
Done in the latest commit.
listeners are kept separate because test fail if these files are imported in the node.